src/test/java/presentation/view/FileInputViewTest.java

```diff
@@ -7,7 +7,7 @@
 import org.junit.jupiter.api.Test;
 
 class FileInputViewTest {
-    @DisplayName("fileInputView에서 파일을 읽어서 Map으로 변환한다")
+    @DisplayName("csv 파일을 읽어서 Map으로 변환한다")
```

Comment:
Map으로 변환하는것은 맞지만
어떤 데이터로 변환하는지가 더 중요할 것 같아요!

```suggestion
    @DisplayName("출석 기록 파일을 읽어서 출석 데이터를 반환한다")
```